### PR TITLE
Updating sidebar

### PR DESCRIPTION
**Pull request by Jeevan1**

### CODE REVIEW

File: src/components/AppBar.jsx [hunk 1]
```diff
@@ -23,83 +22,118 @@ import { Link } from 'react-router-dom';
...
+              ? `calc(100% - ${drawerWidth}px)`
+              : "100%",
+          marginLeft: "auto",
+        }}
+      >
+        <div
+          style={{
+            display: "flex",
+            alignItems: "center",
+            gap: "8px",
```
Comment (RahulRimal) [inline src/components/AppBar.jsx:91]: use theme

File: src/components/AppBar.jsx [hunk 2]
```diff
@@ -23,83 +22,118 @@ import { Link } from 'react-router-dom';
...
         </div>
-    );
-}
-
+        <div style={{ display: "flex", alignItems: "center" }}>
+          <IconButton onClick={toggleTheme}>
+            <IoPersonSharp className="icon" />
+          </IconButton>
+          <Badge badgeContent={2}>
+            <IconButton onClick={() => console.log("clicked")}>
```
Comment (RahulRimal) [inline src/components/AppBar.jsx:111]: remove the log

File: src/components/Drawer.jsx [hunk 3]
```diff
@@ -1,46 +1,135 @@
...
-transition: width 0.3s ease;
-background-color: ${({ theme }) => theme.palette.background.default};
+  border-right: 1px solid ${({ theme }) => theme.palette.border.primary};
+  transition: width 0.3s ease;
+  background-color: ${({ theme }) => theme.palette.background.default};
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: calc(100vh - ${({ theme }) => theme.sizing.s52});
+  /* position: fixed;
```
Comment (RahulRimal) [inline src/components/Drawer.jsx:29]: remove commented code

File: src/components/Drawer.jsx [hunk 4]
```diff
@@ -1,46 +1,135 @@
...
+            <ListItem prefix={<MdCreditScore />} />
+            <ListItem prefix={<MdOutlineCreditCardOff />} />
+          </>
+        ) : (
+          <>
+            <ListItem
+              prefix={<MdPeople />}
+              text="Users"
+              onClick={() => {
+                console.log("he");
```
Comment (RahulRimal) [inline src/components/Drawer.jsx:52]: remove log

File: src/components/Drawer.jsx [hunk 5]
```diff
@@ -1,46 +1,135 @@
...
+              }}
+            />
+            <ListItem
+              prefix={<FaUsersSlash />}
+              text="Blacklisted Users"
+              onClick={() => {
+                console.log("he");
```
Comment (RahulRimal) [inline src/components/Drawer.jsx:59]: remove log

File: src/components/Drawer.jsx [hunk 6]
```diff
@@ -1,46 +1,135 @@
...
+              }}
+            />
+            <Divider />
+            <ListItem
+              prefix={<MdCreditCard />}
+              text="Loans"
+              onClick={() => {
+                console.log("he");
```
Comment (RahulRimal) [inline src/components/Drawer.jsx:67]: remove log

File: src/components/Drawer.jsx [hunk 7]
```diff
@@ -1,46 +1,135 @@
...
+              }}
+            />
+            <ListItem
+              prefix={<MdCreditScore />}
+              text="Healthy Loans"
+              onClick={() => {
+                console.log("he");
```
Comment (RahulRimal) [inline src/components/Drawer.jsx:74]: remove log

File: src/components/Drawer.jsx [hunk 8]
```diff
@@ -1,46 +1,135 @@
...
+              }}
+              suffix={<FaAngleDown />}
+            />
+            <ListItem
+              prefix={<MdOutlineCreditCardOff />}
+              text="Defaulted Loans"
+              onClick={() => {
+                console.log("he");
```
Comment (RahulRimal) [inline src/components/Drawer.jsx:82]: remove log

File: src/components/Drawer.jsx [hunk 9]
```diff
@@ -1,46 +1,135 @@
...
+            <Divider />
+            <ListItem prefix={<MdExitToApp />} />
+          </>
+        ) : (
+          <>
+            <ListItem
+              prefix={<MdOutlineSettings />}
+              text="Settings"
+              onClick={() => {
+                console.log("he");
```
Comment (RahulRimal) [inline src/components/Drawer.jsx:102]: remove log

File: src/components/Drawer.jsx [hunk 10]
```diff
@@ -1,46 +1,135 @@
...
+              }}
+            />
+            <ListItem
+              prefix={<MdOutlinePrivacyTip />}
+              text="Security and Privacy"
+              onClick={() => {
+                console.log("he");
```
Comment (RahulRimal) [inline src/components/Drawer.jsx:109]: remove log

File: src/components/Drawer.jsx [hunk 11]
```diff
@@ -1,46 +1,135 @@
...
+                  icon={
+                    <MdExitToApp
+                      style={{ fontSize: theme.typography.fontSize.f20 }}
+                    />
+                  }
+                />
+              }
+              text="Log Out"
+              onClick={() => {
+                console.log("he");
```
Comment (RahulRimal) [inline src/components/Drawer.jsx:125]: remove log

File: src/components/ListItem.jsx [hunk 12]
```diff
@@ -1,41 +1,51 @@
...
-    font-size: 24px;
-    cursor: pointer;
-    &:hover{
-      background-color: ${({ theme }) => theme.palette.background.dark};
-    }
+  list-style: none;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 12px;
```
Comment (RahulRimal) [inline src/components/ListItem.jsx:15]: use theme here

File: src/components/ListItem.jsx [hunk 13]
```diff
@@ -1,41 +1,51 @@
...
+  font-size: 24px;
```
Comment (RahulRimal) [inline src/components/ListItem.jsx:16]: use theme here

**Review by RahulRimal — CHANGES_REQUESTED** [review]

File: src/components/AppBar.jsx [hunk 14]
```diff
@@ -23,83 +22,118 @@ import { Link } from 'react-router-dom';
...
-justify-content: space-between;
-align-items: center;
-background-color: ${({ theme }) => theme.palette.background.default};
-transition: width  0.3s ease;
-border-bottom: 1px solid ${({ theme }) => theme.palette.border.primary};
-/* border-top-right-radius: 20px; */
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  /* padding: ${({ theme }) => theme.spacing.s12}; */
```
[the remark this answers is not in the document]
Comment (Jeevan1) [reply src/components/AppBar.jsx:42]: done

File: src/components/AppBar.jsx (hunk 2 above)
Comment (Jeevan1) [reply src/components/AppBar.jsx:111]: done

File: src/components/Drawer.jsx (hunk 3 above)
Comment (Jeevan1) [reply src/components/Drawer.jsx:29]: done

File: src/components/Drawer.jsx (hunk 4 above)
Comment (Jeevan1) [reply src/components/Drawer.jsx:52]: done

File: src/components/Drawer.jsx (hunk 5 above)
Comment (Jeevan1) [reply src/components/Drawer.jsx:59]: done

File: src/components/Drawer.jsx (hunk 4 above)
Comment (Jeevan1) [reply src/components/Drawer.jsx:52]: done

File: src/components/Drawer.jsx (hunk 3 above)
Comment (Jeevan1) [reply src/components/Drawer.jsx:29]: done

File: src/components/Drawer.jsx (hunk 7 above)
Comment (Jeevan1) [reply src/components/Drawer.jsx:74]: done

File: src/components/Drawer.jsx (hunk 6 above)
Comment (Jeevan1) [reply src/components/Drawer.jsx:67]: done

File: src/components/Drawer.jsx (hunk 8 above)
Comment (Jeevan1) [reply src/components/Drawer.jsx:82]: done

File: src/components/Drawer.jsx (hunk 9 above)
Comment (Jeevan1) [reply src/components/Drawer.jsx:102]: done

File: src/components/Drawer.jsx (hunk 10 above)
Comment (Jeevan1) [reply src/components/Drawer.jsx:109]: done

File: src/components/Drawer.jsx (hunk 11 above)
Comment (Jeevan1) [reply src/components/Drawer.jsx:125]: done

File: src/components/ListItem.jsx (hunk 13 above)
Comment (Jeevan1) [reply src/components/ListItem.jsx:16]: done

File: src/components/ListItem.jsx (hunk 12 above)
Comment (Jeevan1) [reply src/components/ListItem.jsx:15]: done

File: src/components/AppBar.jsx [hunk 15]
```diff
@@ -23,83 +22,118 @@ import { Link } from 'react-router-dom';
...
+  background-color: ${({ theme }) => theme.palette.background.default};
+  transition: width 0.3s ease;
+  border-bottom: 1px solid ${({ theme }) => theme.palette.border.primary};
+  /* border-top-right-radius: 20px; */
```
[the remark this answers is not in the document]
Comment (Jeevan1) [reply src/components/AppBar.jsx]: done

[the remark this answers is not in the document]
Comment (Jeevan1) [reply src/components/AppBar.jsx]: done

File: src/components/AppBar.jsx (hunk 1 above)
Comment (Jeevan1) [reply src/components/AppBar.jsx]: done

**Review by RahulRimal — APPROVED** [review]

**Pull request merged**